From 1abedad2795b13e9b94dd2b9397f6063e0f8256e Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 24 Feb 2022 18:14:15 +0100
Subject: [PATCH 326/529] ARM: dts: stm32: add vrefint calibration on stm32mp13

Describe vrefint calibration cell to be retrieved through bsec,
on STM32MP13x SoCs family.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/283313
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Ic448c6e25274e7b607659f74619d7e23df8c026c
---
 arch/arm/boot/dts/stm32mp131.dtsi | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm/boot/dts/stm32mp131.dtsi
+++ b/arch/arm/boot/dts/stm32mp131.dtsi
@@ -1536,6 +1536,9 @@
 				reg = <0x4 0x2>;
 				bits = <0 12>;
 			};
+			vrefint: vrefin-cal@52 {
+				reg = <0x52 0x2>;
+			};
 			ts_cal1: calib@5c {
 				reg = <0x5c 0x2>;
 			};
